### ISSUE TITLE
Pretrained Model for VQA

### ISSUE THREAD

**Issue by ojus1** [airsplay/lxmert#7]
If possible, can you guys release a fine tuned snapshot of the model on VQA dataset? 

Thanks.

**Comment by airsplay** [comment]
Hi @ojus1 ,

I uploaded a pre-trained VQA snapshot [here](https://drive.google.com/file/d/1L2TM9Y_KZsl4x28CJe9H82_P39gGyhTe/view?usp=sharing). The "minival" accuracy is about 70.1. Please kindly read [this]https://github.com/airsplay/lxmert/issues/5) if you are confused about the difference.
